less/home-page.less

```diff
@@ -56,6 +83,59 @@
     .destination {
       width: 30%;
       margin-bottom: 30px;
+      .btn{
+        .custom-button(30px, 100%, @button-bg, @white, 10px);
+      }
     }
+
+    @media @mobile{
+      flex-direction: column;
+      align-items: center;
+      .destination{
+        width: 80%;
+      }
+    }
+  }
+
+  .form {
```

Comment:
Contact form looks great, but I would consider putting it in the 'contact' page, for the sake of clarity and convention.